Source/Editor/CMakeLists.txt

```diff
@@ -27,6 +27,43 @@ endif ()
 file (GLOB_RECURSE SOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.cpp *.h *.hpp)
 list (REMOVE_ITEM SOURCE_FILES Editor.cpp EditorWrapper.cpp)
 
+#
+# Groups sources so that they're displayed in IDE matching their layout in the filesystem
+#
+file (GLOB CoreFiles ${CMAKE_CURRENT_SOURCE_DIR} Core/*.*)
```

Comment:
@rokups do we not have a macro that does this kind of stuff automatically?

Reply:
Indeed we do! Simply call `group_sources()` and it will create groups from directories current file is in.